src/main/java/graphql/schema/idl/SchemaGeneratorHelper.java

```diff
@@ -49,9 +56,13 @@ public Object buildValue(Value value, GraphQLType requiredType) {
         Object result = null;
         if (requiredType instanceof GraphQLNonNull) {
             requiredType = ((GraphQLNonNull) requiredType).getWrappedType();
+            assertNotNull(value, "A AST value is required to be present for non null type ;" + requiredType.getName() + "'");
```

Comment:
This assertion throws based on the following SDL:

```graphql
directive @test(include: Boolean!) on FIELD
```

Whereas the following passes:

```graphql
directive @test(include: Boolean! = false) on FIELD
```

Both definitions are valid according to the grammar. Can you elaborate on the thinking behind adding this assertion?

Reply:
@bbakerman ?

Reply:
Looking into this....